core/src/validator.rs

```diff
@@ -487,13 +487,12 @@ impl Validator {
         let id = identity_keypair.pubkey();
         assert_eq!(&id, node.info.pubkey());
 
-        warn!("identity: {}", id);
-        warn!("vote account: {}", vote_account);
+        info!("identity pubkey: {id}");
+        info!("vote account pubkey: {vote_account}");
```

Comment:
is _suspect_ that these were `warn!()` so that they'd still print when operating with info logging disabled.  not sure if that's necessarily a problem or not.  most likely points to the logging being in the wrong place. should probably always init logging in info, resolve cli args and print critical configuration stuff like this, then apply user-requested log level

Reply:
>is suspect that these were warn!() so that they'd still print when operating with info logging disabled

Agreed that something like this is the probable explanation.

> most likely points to the logging being in the wrong place.

At the very least, local-cluster tests benefit from having these logs in `Validator::new()` as opposed to `validator/src/main.rs`.

> should probably always init logging in info, resolve cli args and print critical configuration stuff like this, then apply user-requested log level

Yeah, something like that could make sense